hkmc2/shared/src/test/mlscript/bbml/bbGetters.mls

```diff
@@ -1,8 +1,20 @@
 :js
 :noSanityCheck // * For some reason, these cause problems
-
```

Comment:
Bad idea.

(The way BbML is set up is very hacky and this needs to be done this way for now.)

Reply:
I didn't tend to do that, just didn't review the diff for bbml because of the T and upside down T problem; plan to investigate it today.